run.js

```diff
@@ -60,13 +60,27 @@ function getHistoryCommits(from, to) {
   })
 }
 
+function getOptsFromConfig(config) {
+  return {
+    parserOpts:
+      config.parserPreset != null && config.parserPreset.parserOpts != null
+        ? config.parserPreset.parserOpts
+        : {},
+    plugins: config.plugins != null ? config.plugins : {},
```

Comment:
I found it weird that `@commitlint/cli` defaults `plugins` (which is an array) to `{}`, but if they do that then I think it makes sense to keep same behavior :sweat_smile: